test/components/NavBar/NavBar.test.jsx

```diff
@@ -30,31 +30,31 @@ describe('resize tests', () => {
   it('renders NavbarDesktop when user is logged in on larger screen device', () => {
     window.matchMedia = createMatchMedia(1200);
 
-    const { getByLabelText } = render(
+    const { getByRole } = render(
       <SessionContext.Provider value={{ session: { info: { isLoggedIn: true } } }}>
         <BrowserRouter>
           <NavBar />
         </BrowserRouter>
       </SessionContext.Provider>
     );
 
-    const iconMenu = getByLabelText('menu');
+    const iconMenu = getByRole('group');
```

Comment:
Is there something that by default has a role of group? I'm not seeing anything directly expressed as such.

This is a learning question for me, not questioning this logic.

Reply:
Unfortunately, nothing has a `group` role by default. `div`'s are often used for grouping, but are otherwise devoid of any native `role`.